frontend/src/components/VCollectionHeader/meta/VCollectionHeader.stories.mdx

```diff
@@ -0,0 +1,111 @@
+import {
+  ArgsTable,
+  Canvas,
+  Description,
+  Meta,
+  Story,
+} from "@storybook/addon-docs"
+
+import VCollectionHeader from "~/components/VCollectionHeader/VCollectionHeader.vue"
+
+export const Template = (args) => ({
+  template: `<div class="wrapper w-full p-2"><VCollectionHeader v-bind="args"/></div>`,
+  components: { VCollectionHeader },
+  setup() {
+    const collectionId = args.id || "tag"
+    const defaultCollectionArgs = collections.find((c) => c.id === collectionId)
+    for (let key of ["collection", "title", "url", "resultsLabel"]) {
+      if (args[key] === undefined) {
+        args[key] = defaultCollectionArgs[key]
+      }
+    }
+    return { args }
+  },
+})
+
+export const AllCollectionsTemplate = (args) => ({
+  template: `
+<div class="wrapper w-full p-2 gap-2">
+    <div v-for="collection in args.collections" :key="collection.collection" class="pb-6">
+    <h2 class="heading-3 pb-2">{{ collection.collection }}</h2>
+    <div class="p-10">
+        <VCollectionHeader v-bind="collection" class="bg-white"/>
+     </div>
+    </div>
+</div>`,
+  components: { VCollectionHeader },
+  setup() {
+    return { args }
+  },
+})
+
+<Meta
+  title="Components/VCollectionHeader"
+  component={VCollectionHeader}
+  argTypes={{
+    id: {
+      options: ["tag", "source", "creator", "source-with-long-name"],
+      control: { type: "select" },
+    },
```

Comment:
`id` is a confusing name for this. I didn't understand what it did in the story controls until reading the code. `example-collection` would be clearer. Note: changing the name here requires updating the `Template` story implementation as well.

I wonder if we need two different stories, though? The all collections story feels entirely appropriate for the default story. It isn't a huge component that is a drag to use if all the examples are rendered at a single time.